_data/simple-icons.json

```
@@ -2660,6 +2660,11 @@
             "hex": "258AAF",
             "source": "http://www.sitepoint.com"
         },
+        {
+            "title": "Skillshare",
+            "hex": "17C5CB",
+            "source": "https://www.skillshare.com/press"
```

I think the link you provided in the PR description is more useful here, as I'm not even seeing the logo on this page.

```suggestion
            "source": "https://company-89494.frontify.com/d/Tquwc3XMiaBb/skillshare-style-guide"
```

---

Updated it with your suggestion.